feature-libs/product-configurator/rulebased/root/variant/variant-configurator-overview-layout.module.ts

```diff
@@ -21,20 +21,7 @@ import { VariantConfiguratorPageLayoutHandler } from './variant-configurator-pag
       layoutSlots: {
```

Comment:
@Larisa-Staroverova I know that this is not directly related to this file, but: We need to ensure that the OV page looks nice also if the new sidebar slot is not available (older commerce release!). So the solution we have here and in feature-libs/product-configurator/rulebased/styles/pages/_configurator-variant-overview-page.scss is not sufficient